Gemfile

```diff
@@ -3,13 +3,21 @@ source ENV['GEM_SOURCE'] || "https://rubygems.org"
 group :development, :test do
   gem 'rake'
   gem 'puppetlabs_spec_helper', :require => false
-  gem 'beaker',                 :require => false
-  gem 'beaker-rspec',           :require => false
-  gem 'serverspec',             :require => false
   gem 'rspec-puppet', '~> 1.0'
   gem 'puppet-lint',  '~> 1.1'
 end
 
+group :system_tests do
+  case RUBY_VERSION
+  when '1.8.7'
```

Comment:
I wouldn't even bother doing this, like I was saying last week, if beaker needs Ruby 1.9.3, then we should never run it on anything less. Be mindful that this versioning has nothing to do with our supported Ruby versions, since these are for acceptance tests, not for unit testing.